README.md

```diff
@@ -1,11 +1,22 @@
-# OpenFisca Web-API
+# OpenFisca Web-API - DEPRECATED
 
-[![Build Status](https://travis-ci.org/openfisca/openfisca-web-api.svg?branch=master)](https://travis-ci.org/openfisca/openfisca-web-api)
+## This package is deprecated
 
-[More build statuses](https://www.openfisca.fr/build-status)
+**The official API is now available in [OpenFisca Core repository](https://github.com/openfisca/openfisca-core).**  
+**The current package isn't maintained anymore: it won't evolve, and it is not guaranteed to be compatible with future versions of OpenFisca.**
```

Comment:
Hum, this package is still used by some of our main re-users. We should try to find a balance in order to incentivize them to use the new API, without scaring them too much. I'd remove the "it won't evolve" part.

Reply:
🙂